wordle-web/src/assets/main.css

```diff
@@ -0,0 +1,24 @@
+body {
+  min-height: 100vh;
+  color: white;
+  background: #1c232b;
+  transition: color 0.5s, #1c232b 0.5s;
+  line-height: 1.6;
+  font-family: Inter, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu,
+    Cantarell, 'Fira Sans', 'Droid Sans', 'Helvetica Neue', sans-serif;
+  font-size: 15px;
+  text-rendering: optimizeLegibility;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+#app {
```

Comment:
It's cool you got rid of the two column layout. I had been trying to figure out how to do it and for some reason didn't even notice the `main.css` file until I was looking over your code.